### DISCUSSION TITLE
Default View Changed - How to Expand Parts and Stock By Default

### DISCUSSION THREAD

**Discussion by imager2019** [inventree/InvenTree#8059] [General]
Recently updated to v0.16.1 and find my Parts and Stock default displays have now changed.
Previous version that we had running (0.15.4) when you select the Top Menu bar Parts or Stock options, the Stock Items or Parts Items would display.

With the version 16.1, we only see the following:
![image](https://github.com/user-attachments/assets/b2fc87be-6373-48d0-b2a8-81df3debf4c8)

Not sure if this is a default setting that has changed or become corrupt in the upgrade or not. We were however expecting the same expanded style as before (as is how the demo site displays).

![image](https://github.com/user-attachments/assets/1116a722-9d36-436e-b84b-ada197e03be7)

Is this a setting that we have incorrectly configured or a but in moving to the 16.1 version?

Thoughts??

**Comment by SchrodingersGat** [comment]
This looks like potentially a caching issue in your browser: https://docs.inventree.org/en/stable/faq/#feature-x-does-not-work-after-update

**Reply by imager2019** [reply]
I thought about that too and did the cache clearing first. Also tried on other machines which had previous not connected to the site in the past with the same result along with other browsers.

The default page displays properly, but I do notice that the Alert Bell is showing a Server Error {Error Object (57)}
![image](https://github.com/user-attachments/assets/1a25e47c-3345-4002-9749-cd7b67a49033)

**Reply by SchrodingersGat** [reply]
Ok, can you click on the error notification and share what the message is? It might provide some good insights.

**Reply by imager2019** [reply]
I tried clicking on the error notification, but the system times out and never displays it.

![image](https://github.com/user-attachments/assets/1242179f-477d-4ca4-bdb4-97efb4d15b01)

I did notice that the error page that comes up (from the web browser), says that the site cannot be reached as it took too long to respond. The IP address that comes up is my Public IP, not the IP of the machine running Inventree (which is 192.168.1.222). 
![image](https://github.com/user-attachments/assets/d232418d-787d-423d-ba9f-771a0ffea3cb)

I reviewed the settings under Server Settings and see that my Base URL defaults to my Public IP address (see below, with it partially blurred out).
![image](https://github.com/user-attachments/assets/f4b83671-528d-4fcf-92f8-a17c2ca927a6)


I cannot recall that this was the case previous to the upgrade.

Is there any other testing I can do to assist in diagnosing this issue?

John

**Comment by imager2019** [comment]
I performed some additional testing this evening and seem to have it working now.
I ran the `inventree run cli update`

This took about a minute to run through and it came back and when I revisit the site, the Parts and Stock menus are working as they should now. I will continue to do some testing, but so far, it looks promising.

Could you clarify what the correct procedure is to update a site?
I had previously run:
`apt upgrade inventree`
`inventree run cli migrate` - This was run to correct the Database Migration issue and messages
`inventree run cli update` - This was executed today and seem to have fixed the issues

Is this the correct order and process to update a site?

John

**Reply by matmair** [reply]
upgrading the package should be the only thing necessary. Do you have the logs from the last time updating? They should be in `/opt/inventree/log/`

**Comment by SchrodingersGat** [comment]
@imager2019 yes the `update` function must be performed when updating to the latest version. If that has fixed the issues for you, it sounds like a good outcome!

**Comment by imager2019** [comment]
Attached is the log file which is from Sept. 2 upgrade.

See attached..

Thanks.. John

**Reply by matmair** [reply]
@imager2019 please upload the file directly to the Git Hub discussion. Appending as a mail to GitHub is not working

**Reply by imager2019** [reply]
[setup_2024-09-02_07_09_24.log.zip](https://github.com/user-attachments/files/16882998/setup_2024-09-02_07_09_24.log.zip)
@matmair .. Attached is the log file uploaded directly here.. Thanks...

**Reply by matmair** [reply]
Thank you for the log @imager2019 - it seems like the update ran before we released the patched `0.16.1`. The update did not fully run through - leaving the migrations unfinished. Please run `apt update && apt upgrade inventree` again to make sure the latest version is installed.

**Reply by imager2019** [reply]
Thanks @matmair .. Just ran the command to update to make sure the latest is installed...
Appreciate the assistance!
